src/utils/config-loader.js

```diff
@@ -0,0 +1,82 @@
+var fs = require( 'fs' );
+var path = require( 'path' );
+var yaml = require( 'js-yaml' );
+var merge  = require('lodash.merge');
+var defaultOptions = require( '../default-options' );
+var utils = require( './utils' );
+var  C = require( '../constants/constants' );
+var argv = require( 'minimist' )( process.argv.slice(2) );
+
+module.exports = function(customFilePath) {
+  var order = [
+    'config.json',
+    'config.js',
+    'config.yml',
+  ];
+  if ( customFilePath != null ) {
+    order.unshift(customFilePath)
+  }
+
+  var filePath = order.filter(function( filePath ) {
+    try {
+      fs.lstatSync(filePath)
+      return true
+    } catch ( err ) {}
+  })[0]
+
+  if ( filePath == null ) {
+    return defaultOptions.get();
```

Comment:
I think it might make sense to throw an error if no file could be found. Otherwise you might thing that your confix.json file is used, but due to typo it isn't